packages/synapse-react-client/src/components/CreateTableViewWizard/CreateTableViewWizard.integration.test.tsx

```diff
@@ -49,6 +50,24 @@ const createEntitySpy = jest.mocked(SynapseClient.createEntity)
 const getAnnotationColumnsSpy = jest.mocked(
   SynapseClient.getAnnotationColumnModels,
 )
+
+const defaultColumnModelsWithoutId = defaultFileViewColumnModels.map(cm =>
+  omit(cm, ['id']),
+)
+const annotationColumnModelsWithoutId = MOCK_ANNOTATION_COLUMNS.results.map(
+  cm => omit(cm, ['id']),
+)
+
+async function getLatestCreatedColumnModelIdsFromSpy(
+  spy: typeof createColumnModelsSpy,
+) {
+  return (
+    await (spy.mock.results[0].value as ReturnType<
+      typeof SynapseClient.createColumnModels
+    >)
+  ).list.map(cm => cm.id)
+}
```

Comment:
Get the IDs of the new column models returned by the most recent call to `createColumnModels`. We'll verify that these IDs get passed to another function.